torchaudio/csrc/rnnt/cpu/compute.cpp

```diff
@@ -116,6 +116,18 @@ std::tuple<torch::Tensor, c10::optional<torch::Tensor>> compute(
       /*int_size=*/int_workspace.numel());
 
   switch (logits.scalar_type()) {
+      case torch::ScalarType::Double: {
+      Compute</*DTYPE=*/double, /*CAST_DTYPE=*/float>(
```

Comment:
from offline discussion, we need to confirm that CAST_TYPE controls the return type. if that is the case, we'll need to add a test to validate that the type matches the one requested by the user instead.

Reply:
in our case here, it means that gradcheck may only still be receiving float32 and not float64. can you verify that?

Reply:
yea, looks like it's still float32

Reply:
can you confirm what the results are for numpy with float64?